src/components/OcModal.vue

```diff
@@ -13,28 +13,30 @@
           v-else-if="hasInput"
           key="modal-input"
           ref="ocModalInput"
-          v-model="$_ocModal_input_value"
+          v-model="input_value"
```

Comment:
`inputValue` or even `input`?

The same goes for some more properties in this component. I get the intention of a prefix for the input related stuff, but it is already kinda mixed up all over the place... So I vote for omitting the prefix, or at least keep it camelCase :)

Reply:
`inputValue` (camel case please :-) )

Reply:
Should be resolved now!